members/SPAU3393804d.yaml

```diff
@@ -1,69 +1,65 @@
 bioguide: SPAU3393804d
 contact_form:
   method: post
-  action:
   steps:
-    - visit: "https://wpcontact.pasenategop.com/contact.aspx?websiteid=28"
-    - wait:
-        - value: 2
+    - visit: "https://contact1.pasenategop.com/contact/28"
+    - find:
+        - selector: "#ContactForm_FirstName"
     - fill_in:
         - name: First Name
-          selector: input#txtFirstname
+          selector: "#ContactForm_FirstName"
           value: $NAME_FIRST
           required: true
+          options:
+            max_length: 30
         - name: Last Name
-          selector: "#txtLastname"
+          selector: "#ContactForm_LastName"
           value: $NAME_LAST
           required: true
+          options:
+            max_length: 50
         - name: email
-          selector: "#txtEmail"
+          selector: "#ContactForm_EmailAddress"
           value: $EMAIL
           required: true
         - name: phone
-          selector: "#txtPhone"
-          value: $PHONE_PARENTHESES
+          selector: "#ContactForm_HomePhone"
+          value: $PHONE
           required: true
-        - name: Address
-          selector: "#txtAddress1"
+        - name: address
+          selector: "#ContactForm_Address1"
           value: $ADDRESS_STREET
           required: true
+          options:
+            max_length: 100
         - name: address2
-          selector: "#txtAddress2"
+          selector: "#ContactForm_Address2"
           value: $ADDRESS_STREET_2
           required: false
-        - name: City
-          selector: "#txtCity"
+        - name: city
+          selector: "#ContactForm_City"
           value: $ADDRESS_CITY
           required: true
+          options:
+            max_length: 40
         - name: zip
-          selector: "#txtZip"
+          selector: "#ContactForm_ZipCode"
           value: $ADDRESS_ZIP5
           required: true
-        - name: Subject
-          selector: "#txtSubject"
-          value: $SUBJECT
-          required: false
-        - name: Message
-          selector: "#txtComments"
+        - name: message
+          selector: "#ContactForm_Comments"
           value: $MESSAGE
           required: true
-    - find:
-        - selector: div#ddlIssue a#ddlIssue_Arrow
-    - click_on:
-        - selector: div#ddlIssue a#ddlIssue_Arrow
-    - wait:
-        - value: 1
-    - find:
-        - selector: div#ddlIssue_DropDown li:nth-child(2)
-    - click_on:
-        - selector: div#ddlIssue_DropDown li:nth-child(2)
+    - select:
+        - name: Topic
+          selector: "#issueselect"
+          value: Legislation
+          required: true
```

Comment:
spot checked the 2 forms without newsletter default opt-in, looks like you need to opt out on this form too

Reply:
good call, thanks for checking!